f5/bigip/sys/db.py

```diff
@@ -32,16 +32,28 @@ def __init__(self, dbs):
         self._meta_data['required_json_kind'] = 'tm:sys:db:dbstate'
```

Comment:
So is "`Db`" a class that should inherit from "`UnnamedResourceMixin`"?   It seems like it probably is.  I'm guessing it was defined before the mixin.